platform/viewer/src/lib/DICOMSR/retrieveDataFromMeasurements.js

```diff
@@ -0,0 +1,34 @@
+import * as dcmjs from 'dcmjs';
+import OHIF from '@ohif/core';
+
+const retrieveDataFromMeasurements = measurements => {
+  const { MeasurementReport } = dcmjs.adapters.Cornerstone;
+  const { getImageIdForImagePath } = OHIF.viewerbase;
```

Comment:
OHIF.viewerbase doesn't seem to have this function in my environment. Maybe it should be OHIF.measurements instead?

Reply:
You are correct. 
At the momment I've create the draft PR I haven't tested store functionality.

Reply:
Store is working pretty well. I've been trying it out. :)